src/Loop/Driver.php

```diff
@@ -15,7 +15,13 @@
     private $registry = [];
 
     /**
-     * Start the event loop.
+     * Run the event loop.
+     *
+     * One iteration of the loop is called one "tick". A tick covers the following steps:
+     *
+     *  1. Activate watchers created / enabled in the last tick / before `run()`.
+     *  2. Execute all enabled defer watchers.
+     *  3. Execute all due timer and actionable stream callbacks, each only once per tick.
```

Comment:
Can signals be executed at any time or should they be included in this step? All current implementations execute pending signals only in this step, but should that be part of the spec?